config/environments/production.rb

```diff
@@ -41,7 +41,7 @@
   config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
 
   # Prevent health checks from clogging up the logs.
-  config.silence_healthcheck_path = "/up"
+  config.silence_healthcheck_path = /^\/status/
```

Comment:
AFAICT, most of our other boxes use:


```suggestion
  config.silence_healthcheck_path = '/status'
```

H3 is an outlier here.